entities/src/main/java/com/flipkart/varadhi/entities/AbstractTopic.java

```diff
@@ -1,6 +1,9 @@
 package com.flipkart.varadhi.entities;
 
 public class AbstractTopic extends VaradhiResource {
+
+    public static String NAME_SEPARATOR = ".";
```

Comment:
Better to move to VaradhiResource, other entities likely to have similar requirement e.g. subscription/queue.